ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionRequestModal.tsx

```diff
@@ -28,35 +30,54 @@ export type ExceptionRequestModalOptions = {
 
 export type ExceptionRequestModalProps = {
     type: CveExceptionRequestType;
+    isUpdate?: boolean;
+    id?: string;
     cves: ExceptionRequestFormProps['cves'];
     scopeContext: ScopeContext;
-    onExceptionRequestSuccess: (vulnerabilityException: BaseVulnerabilityException) => void;
+    onExceptionRequestSuccess: (vulnerabilityException: VulnerabilityException) => void;
     onClose: () => void;
 };
 
 function ExceptionRequestModal({
     type,
+    isUpdate = false,
+    id = '',
     cves,
     scopeContext,
     onExceptionRequestSuccess,
     onClose,
 }: ExceptionRequestModalProps) {
     const cveCountText = pluralize(cves.length, 'workload CVE');
+    const titleAction = isUpdate ? 'Update' : 'Request';
     const title =
         type === 'DEFERRAL'
-            ? `Request deferral for ${cveCountText}`
-            : `Mark ${cveCountText} as false positive`;
+            ? `${titleAction} deferral for ${cveCountText}`
+            : `${titleAction} false positive for ${cveCountText}`;
```

Comment:
I slightly changed the wording for the false positive title because it was easier to construct a sentence that worked for both creating a request and updating a request